src/python/pants/backend/jvm/subsystems/zinc_language_mixin.py

```diff
@@ -4,12 +4,18 @@
 
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from builtins import object
+from pants.build_graph.mirrored_target_option_mixin import MirroredTargetOptionMixin
 
 
-class ZincLanguageMixin(object):
+class ZincLanguageMixin(MirroredTargetOptionMixin):
   """A mixin for subsystems for languages compiled with Zinc."""
 
+  mirrored_target_option_actions = {
+    'strict_deps': lambda tgt: getattr(tgt, 'strict_deps'),
```

Comment:
Why `getattr` rather than just `tgt.strict_deps`? If the intent is to default to returning `None` if the field doesn't exist, I think that you'll need to pass the third argument to `getattr`.

Reply:
Fixed to just use `tgt.strict_deps`!